README.md

```diff
@@ -1,25 +1,196 @@
-# Document Format: 
+# 👋 Welcome! 
+This is the community-maintained documentation for [Bloom.host](https://bloom.host)!
+
+# 🌿 Contributing
+
+<details>
+
+<summary>
+
+## 📚 To The Documentation
+
+</summary>
+
+You can help contribute to the Bloom.host documentation in one of two ways:
+
+### Option #1:
+
+You can write an article in a format you prefer and create a support ticket in
+our [Discord server](https://discord.gg/bloom), and it will be converted over for you. 
+Note that documents in Markdown are preferred, however, Microsoft Word and Google Docs files can be accepted,
+and they can be uploaded to the documentation.
+
+---
+
+### Option #2 (Advanced):
+
+You can also contribute to the docs directly by doing the following:
+
+This method makes it easier to import the document(s) and to make sure that the formatting is correct and to make it
+look the way you want it when it is imported.
+
+1. Create a fork of the [GitHub Repository](https://github.com/Bloom-host/BloomDocs)
+2. Make edits to your fork of this repository.
```

Comment:
Maybe the changes should be made in a separate branch?

Reply:
> Maybe the changes should be made in a separate branch?

I don't think there is a good way to do that on GitHub. Forking is the intended way to create upstream contrib externally from what I know.

Reply:
Ah, sorry. I meant forking then making a new branch for changes. That’s how I’ve always been told to do it

Reply:
Ohh gotcha! Yeah, I added it.